### PR TITLE
feat: add extraFields to ObjectSection

### PR DESCRIPTION
**Pull request by c0state**

### CODE REVIEW

File: packages/graphql-explorer/src/ObjectSection.tsx [hunk 1]
```diff
@@ -123,11 +124,16 @@ function ObjectSectionField({
   );
 }
 
+interface ObjectSectionProps extends SectionProps<Obj, g.GraphQLObjectType> {
+  extraFields: ListGroupItem | ListGroupItem[];
```
Comment (c0state) [inline packages/graphql-explorer/src/ObjectSection.tsx:128]: Tightened up the types are as it currently must be a ListGroupItem @itajaja

File: packages/graphql-explorer/src/ObjectSection.tsx [hunk 2]
```diff
@@ -123,11 +124,16 @@ function ObjectSectionField({
   );
 }
 
+interface ObjectSectionProps extends SectionProps<Obj, g.GraphQLObjectType> {
+  extraFields?: ListGroupItem | ListGroupItem[];
```
Comment (itajaja) [inline packages/graphql-explorer/src/ObjectSection.tsx:128]: nope, unfortunately you can't do that. when you do `<ListGroupItem ... />` this returns a ReactNode, not a ListGroupItem. I am pretty sure that if you apply this change to lando-ui, ts will complain. in general this is a broad limitation of the react type system. everything is a node, or at best an element, but you cannot discern what type of element the element is.

**Review by itajaja — CHANGES_REQUESTED** [review]

File: packages/graphql-explorer/src/ObjectSection.tsx (hunk 2 above)
Comment (c0state) [reply packages/graphql-explorer/src/ObjectSection.tsx:128]: You're right. I noticed this when testing as you suspected. I decided not to use this though for now as it intermingles custom fields with the schema fields. Better to separate them in another component like it was before I think

**Comment by c0state** [comment]
Per discussion, closing. Need to think about how to pass extras field, components, widgets, etc. in a more thought out way first.

**Pull request closed without merging**